Setup/Patch/Data/CustomerCompanyAttribute.php

```diff
@@ -0,0 +1,89 @@
+<?php
+namespace Devall\Company\Setup\Patch\Data;
+
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use Magento\Framework\Setup\Patch\PatchRevertableInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Customer\Setup\CustomerSetupFactory;
+use Magento\Customer\Setup\CustomerSetup;
+
+class CustomerCompanyAttribute implements DataPatchInterface, PatchRevertableInterface
+{
+
+    /**
+     * @var ModuleDataSetupInterface
+     */
+    private $moduleDataSetup;
+    /**
+     * @var CustomerSetup
+     */
+    private $customerSetupFactory;
+
+    /**
+     * Constructor
+     *
+     * @param ModuleDataSetupInterface $moduleDataSetup
+     * @param CustomerSetupFactory $customerSetupFactory
+     */
+    public function __construct(
+        ModuleDataSetupInterface $moduleDataSetup,
+        CustomerSetupFactory $customerSetupFactory
+    ) {
+        $this->moduleDataSetup = $moduleDataSetup;
+        $this->customerSetupFactory = $customerSetupFactory;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function apply() {
+        $this->moduleDataSetup->getConnection()->startSetup();
+        /** @var CustomerSetup $customerSetup */
+        $customerSetup = $this->customerSetupFactory->create(['setup' => $this->moduleDataSetup]);
+        $customerSetup->addAttribute(
+            \Magento\Customer\Model\Customer::ENTITY,
+            'company',
+            [
+                'type' => 'int',
+                'label' => 'Company',
+                'input' => 'text',
+                'source' => '',
+                'required' => false,
+                'visible' => true,
+                'position' => 500,
+                'system' => false,
+                'backend' => ''
+            ]
+        );
+
+        $attribute = $customerSetup->getEavConfig()->getAttribute('customer', 'company')->addData(['used_in_forms' => ['adminhtml_customer']]);
+        $attribute->save();
```

Comment:
Better to use non-deprecated repository method.

Reply:
tried attributeRepository->save() but attribute,s not being saved after i modify in admin customer edit

Reply:
Ok never mind on the attributeRepo here. 
On the other hand why do you even need the ['used_in_forms' => ['adminhtml_customer'] ?

Reply:
tried removed it , attribute,s not being saved after i modify